CMSgov/design-system
# [NO-TICKET] Test forced-colors mode for visual regressions

## Summary

I was working on our component maturity checklist and thinking about how we verify that our components work in forced-colors mode and had a realization. With our new visual regression tests, we...
1. Have a pretty thorough set of testing targets that are no longer constrained to needing to finish within a reasonable amount of time for CI tests (because of our nightly builds)
2. Now have a lot of control over those testing targets

So why couldn't we test forced-colors mode as well?

This also removes some story tests that aren't necessary.

## How to test

1. `yarn test:browser --list` if you want to see that it registers all the tests under the new target
2. `yarn test:browser` to run it locally if you want. This takes a long time. We can actually narrow the tests we run with a `--project` argument, and I'm adding documentation for that
3. I know it's a lot of work, but please take a look at the new screenshots and try to spot problems

## Checklist




### If this is a change to code:


### If this is a change to documentation:
